### PR TITLE
Get rid of deprecated *_filter support

### PR DESCRIPTION
**Pull request by sergey-alekseev**
`*_filter` methods were removed in Rails 5.1, see https://github.com/rails/rails/commit/d7be30e8babf5e37a891522869e7b0191b79b757.
Should've been done as a part of cdfea4c6c1ac8f1a227ce57db6b0f758ca376d28, so this is just a leftover addressed.

BTW `skip_action` added in fc876052476711074e0710f56166d193f9c0fa69 never existed in Rails.
`skip_filter` was renamed to `skip_action_callback` in Rails 4.2 https://github.com/rails/rails/commit/6c5f43bab8206747a8591435b2aa0ff7051ad3de
and then deprecated in Rails 5.1 https://github.com/rails/rails/commit/6976e1da07ac91f5e17c6bc02758122255d94d48.

Basically this commit reverts the implementation to 2010, similar to 83c1f220e6930e4c6e11cfda3d00028d99f34439 but more complete and up-to-date.

### CODE REVIEW

**Review by deivid-rodriguez — APPROVED** [review]
👍

**Review by javierjulio — APPROVED** [review]
Thanks!

**Comment by sergey-alekseev** [comment]
Glad to see this merged. Thanks @javierjulio
